torch_brain/utils/readout.py

```diff
@@ -77,7 +77,7 @@ def prepare_for_readout(
 
     # resolve eval mask
     eval_mask = None
-    eval_interval_key = data.config.get("eval_interval", None)
+    eval_interval_key = readout_config.get("eval_interval", None)
     if eval_interval_key is not None:
```

Comment:
This is off-topic for this PR. But seems like a bug to me, so I'm fixing it here.